org.beeref.BeeRef.json

```
@@ -0,0 +1,51 @@
+{
+    "id": "org.beeref.BeeRef",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "6.3",
+    "sdk": "org.kde.Sdk",
+    "base": "com.riverbankcomputing.PyQt.BaseApp",
+    "base-version": "6.3",
+    "command": "beeref",
+    "separate-locales": false,
+    "finish-args": [
+        "--share=network",
+        "--socket=x11",
+        "--share=ipc",
+        "--filesystem=host"
```

per [this comment](https://github.com/flathub/flathub/pull/3324/files#r917879545) and my manual testing we should enable wayland - it's well supported on Qt. Also you should enable GPU acceleration otherwise Qt complains about it.
```suggestion
        "--filesystem=host",
        "--socket=wayland",
        "--device=dri"
```

---

> we should enable wayland - it's well supported on Qt

This statement isn't true. I have seen many Qt bugs resolved by forcing `xcb` QPA